Src/BootCamp.Chapter/Person.cs

```diff
@@ -0,0 +1,74 @@
+﻿namespace BootCamp.Chapter
+{
+    public class Person
+    {
+        public Person(string firstName, string lastName, int age, float weight, float heigth)
+        {
+            _firstName = firstName;
+            _lastName = lastName;
+            _age = age;
+            _weight = weight;
+            _heigth = heigth;
+        }
+
+        private string _firstName;
+
+        public string GetFirstName()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetFirstName' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5149394)